### DISCUSSION TITLE
Mac availability, and zero for graph

### DISCUSSION THREAD

**Discussion by rKila** [jneilliii/OctoPrint-BedLevelVisualizer#440] [General]
Does anyone know how to install it in mac ? I am running into a problem with libatlas3. On a side note, what is the zero for the graph ? where is it getting it from. I have an ender 3 pro and I am assuming it taking it from the switch position (maybe). BLtouch is way too far off from the numbers I am seeing!

**Comment by jneilliii** [comment]
sorry, but don't have a mac to test, I could suggest to use the current release candidate that removes numpy though, which is related to the problem you are describing. 

zero is real world zero for z, and the 0,0 x/y position is derived from your printer profile settings in OP.

**Reply by rKila** [reply]
Thank you for quick response :). Actually, I got it to work by:
>brew install openblas
>brew install lapack

I figured we need a LA lib.
Can you adjust the zero for z manually in the plugin? I am using bltouch but I am homing on zero z from switch. so my bed is level at some weird number. Or is that not how it should work. I tried many different variations of the offsets (because they effect  the zero in question) but they are all wrong for print quality.

**Reply by jneilliii** [reply]
The plugin has no impact on the actual leveling itself, and only visualizes what your firmware returns. You can use the relative z offset option to have all the numbers returned "flatten" down to put them closer to 0.
